### PR TITLE
Commerce 8505 High Prices break UI

### PR DESCRIPTION
**Pull request by christian-chiappa**

### CODE REVIEW

**Comment by christian-chiappa** [comment]
ci:test:sf

**Comment by christian-chiappa** [comment]
ci:sf

**Review by riccardo-alberti — CHANGES_REQUESTED** [review]
@christian-chiappa we should not reference service directly

File: modules/apps/commerce/commerce-product-definitions-web/build.gradle [hunk 1]
```diff
@@ -23,6 +23,7 @@ dependencies {
 	compileOnly project(":apps:commerce:commerce-product-api")
 	compileOnly project(":apps:commerce:commerce-product-ddm-api")
 	compileOnly project(":apps:commerce:commerce-product-item-selector-api")
+	compileOnly project(":apps:commerce:commerce-product-service")
```
Comment (riccardo-alberti) [inline modules/apps/commerce/commerce-product-definitions-web/build.gradle:26]: @christian-chiappa this should reference product-api

**Review by riccardo-alberti — CHANGES_REQUESTED** [review]

**Comment by christian-chiappa** [comment]
ci:test:sf

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**